tests/data/fields/text_field_test.py

```diff
@@ -88,19 +88,19 @@ def test_index_converts_field_correctly(self):
         t_index = vocab.add_token_to_namespace("t", namespace='characters')
         c_index = vocab.add_token_to_namespace("c", namespace='characters')
 
-        field = TextField(["A", "sentence"], [token_indexers["single id"](token_namespace="words")])
+        field = TextField(["A", "sentence"], [token_indexers["single id"](namespace="words")])
         field.index(vocab)
         # pylint: disable=protected-access
         assert field._indexed_tokens == [[capital_a_index, sentence_index]]
```

Comment:
No - I unified the `namespace` attribute of the `TokenIndexer` so we can use it here: https://github.com/allenai/allennlp/pull/25/files#diff-ad92587fe15aa343c6d63c5f1c06cf5fR88. Your way is another solution but this is better because it more directly links using a `TokenIndexer` with a same vocabulary namespace. Actually we should raise an error somewhere if you try to index a single `TextField` with a `TokenIndexer` using the same vocab namespace. I'll add that.